endpoints/status.go

```diff
@@ -6,9 +6,17 @@ import (
 	"github.com/julienschmidt/httprouter"
 )
 
-// Status is the handler function of the "/status" endpoint
-func Status(w http.ResponseWriter, r *http.Request, ps httprouter.Params) {
-	// We might want more logic here eventually... but for now, we're ok to serve more traffic as
-	// long as the server responds.
-	w.WriteHeader(http.StatusNoContent)
+// NewStatusEndpoint returns a handler which writes the given response when the app is ready to serve requests.
+func NewStatusEndpoint(response string) httprouter.Handle {
```

Comment:
Should we add a test for this function?

Reply:
Yes, you're right. Pushing it in the next commit.